bin/annotate_test_failures

```diff
@@ -100,6 +115,63 @@ def update_annotation(title, list, style, state)
   end
 end
 
+# Given a list of failures, parse list and send a slack notification with the test names in the payload
+#
+def send_slack_notification(slack_webhook, slack_channel, title, list)
+    failing_tests = list.map { |item| "#{item.name} in #{item.classname}" }        
+    assertion_failures_count = list.count
+    test_text = (assertion_failures_count == 1) ? "Test" : "Tests"
+
+    slack_message_payload = {
+      "channel": "#{slack_channel}",
+      "username": "#{ENV['BUILDKITE_PIPELINE_NAME']} Tests Failures",
+      "icon_emoji": ":fire:",
+      "blocks": [
+        {
+          "type": "section",
+          "text": {
+            "type": "mrkdwn",
+            "text": ":warning: *#{assertion_failures_count} #{test_text} Failed in #{ENV['BUILDKITE_PIPELINE_NAME']} - #{title}*"
+          },
+          "accessory": {
+            "type": "button",
+            "text": {
+              "type": "plain_text",
+              "text": "Build",
+              "emoji": true
+            },
+            "value": "build",
+            "url": "https://buildkite.com/#{ENV['BUILDKITE_ORGANIZATION_SLUG']}/#{ENV['BUILDKITE_PIPELINE_SLUG']}/builds/#{ENV['BUILDKITE_BUILD_NUMBER']}##{ENV['BUILDKITE_JOB_ID']}",
```

Comment:
The whole first part of that URL is already also exposed by buildkite as `BUILDKITE_BUILD_URL` env var, so you might as well use this to make this more readable 😉 

```suggestion
            "url": "#{ENV['BUILDKITE_BUILD_URL']}##{ENV['BUILDKITE_JOB_ID']}",
```